src/mod.ts

```diff
@@ -44,6 +47,12 @@ const run = async () => {
     }
   }
 
+  // If only outputting project scripts to stdout, do that and end early
+  if (scripts) {
+    console.log(JSON.stringify(projectScripts()));
```

Comment:
What's the value from making this a function rather than just using the object directly?

Reply:
Easy mocking for tests 😄 . IMO Doing `import('some-module').someExport = myMockFunction (args) { return 'some-value-only-used-in-testing'; };` instead of doing the proxyquire or rewire song-n-dance to stub out top-level imports in other modules is way easier. 
Also: was just following the pattern I saw in other modules in this project.